sdk/services/futures.ts

```diff
@@ -843,4 +843,16 @@ export default class FuturesService {
 			amount.toBN(),
 		]);
 	}
+
+	// This is on an interval of 15 seconds.
+	public async getFuturesTrades(marketKey: FuturesMarketKey, minTs: number, maxTs: number) {
```

Comment:
Is this used anywhere, seems like we only call the `getFuturesTrades` from the generated subgraph queries?

Reply:
It's not used yet. I suspended copying over the hook that it replaces, but we'll use it once that is copied to the SDK.